src/Adapter/Currency/CommandHandler/ToggleExchangeRateAutomatizationHandler.php

```diff
@@ -181,7 +181,6 @@ private function getCronUrl()
     private function createCronJob($cronUrl)
     {
         $cronJobsModule = Module::getInstanceByName('cronjobs');
-        /** @var CronJobs $cronJobsModule */
```

Comment:
I had to remove this because phpstan 0.12.80 refuses this phpDoc

```
 ------ -----------------------------------------------------------------------------
  Line   Adapter/Currency/CommandHandler/ToggleExchangeRateAutomatizationHandler.php
 ------ -----------------------------------------------------------------------------
  185    PHPDoc tag @var for variable $cronJobsModule contains unknown class
         CronJobs.
         💡 Learn more at https://phpstan.org/user-guide/discovering-symbols
 ------ -----------------------------------------------------------------------------

```